### ISSUE TITLE
Softlock if steam_plugin_config.ini is empty

### ISSUE THREAD

**Issue by GreenMan36** [FriendsOfGalaxy/galaxy-integration-steam#147]
**Describe the bug**
I've used this integration on and off for a while now; it hasn't been working in the past few months. Turns out my `steam_plugin_config.ini` was empty for some reason. This caused the plugin to fail every time on launch, reinstall, everything. Since the .ini file is not in the plugin folder itself I didn't delete it before so when I read the logs I noticed it said the file was empty and guessed removing it should fix the issue,

I was unable to reproduce this bug but I think it might be valuable for someone having this issue in the future or to check for this issue in code.

**To Reproduce**
I have been unable to reproduce the bug but you might understand the exact error more than me,
Basically somehow the settings file corrupted between now and when the plugin ran last.

**Expected behavior**
I expected the plugin to have error handling; for example renaming the old settings with a _BACKUP suffix and showing a popup that the config has been reset because of an issue.

**Attached logs**
[Error Logs.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/8404267/Error.Logs.zip)
Note: I deleted my logs before trying to fix the issue myself. Files have been taken out of the Windows trash bin so `RECYCLE BIN plugin-steam-ca27391f-2675-49b1-92c0-896d43afa4f8.log` is older than `plugin-steam-ca27391f-2675-49b1-92c0-896d43afa4f8.log`. My bad.
